src/Components/AlbumCard/AlbumCard.js

```diff
@@ -36,7 +36,7 @@ function AlbumCard ({ title }) {
         <div className='card_genre-container' data-cy='card_genre-container'>
           { displayGenres(album.genres) }
         </div>
-        <button className='card_discogs-link' data-cy='card_discogs-link' >Purchase on Discogs</button>
+        <a href={album.uri}className='card_discogs-link' data-cy='card_discogs-link' >Purchase on Discogs</a>
```

Comment:
Add `target='_blank'` to open this link in a new tab.

Reply:
change made